.travis.yml

```diff
@@ -19,8 +19,8 @@ matrix:
     env: SUITE=ubuntu
   - rvm: 2.3
     env: SUITE=windows-server-2012r2
-  - rvm: 2.3
-    env: SUITE=windows-nano
+  # - rvm: 2.3
+  #   env: SUITE=windows-nano
```

Comment:
Understood. I was pulling from current, but the issue was pre-existing. I was hoping to get this test back up and running, so I'm not too concerned about which product or channel. Just need a working appx.